lib/services/role.go

```diff
@@ -76,6 +76,7 @@ var DefaultImplicitRules = []types.Rule{
 	types.NewRule(types.KindKubernetesCluster, RO()),
 	types.NewRule(types.KindUsageEvent, []string{types.VerbCreate}),
 	types.NewRule(types.KindVnetConfig, RO()),
+	types.NewRule(types.KindSPIFFEFederation, RO()),
```

Comment:
My gut feeling is that these are very similar to the concept of a "Certificate Authority" and thus could belong here in the implicit set. Realistically, if you omit this, then both tbot/tctl will have an incomplete view of what the trust bundle should be, so you'd have to include this on all roles you expect to also generate SVIDs. There's also nothing within these that is considered "sensitive". 

I'm curious if anyone has any points in opposition to this and would prefer this to be explicitly assigned to roles.